packages/animated-circle/index.android.ts

```diff
@@ -1,10 +1,20 @@
-import { Color } from '@nativescript/core';
+import { Color, Property, booleanConverter } from '@nativescript/core';
 import { AnimatedCircleCommon, barColorProperty, rimColorProperty, spinBarColorProperty } from './common';
 
 declare const at;
 
+export const animatedProperty = new Property({
```

Comment:
This is not needed as class has already got a getter and setter for this property.
https://github.com/NativeScript/plugins/blob/main/packages/animated-circle/index.android.ts#L97

Reply:
Hi, if you try the code the getter return always true,

Reply:
I have tried to set animated="false" from the ui but it doesn't work .. only after adding the custom property the value was assigned correctly!

Reply:
> Hi, if you try the code the getter return always true,

I see. It seems that the convertion to boolean returns always true as argument value is always a string despite of the type. We'll need the `Property` register but I'll still request few changes on this one.